routesrv/polling.go

```diff
@@ -56,8 +37,9 @@ type poller struct {
 	editRoute      []*eskip.Editor
 	cloneRoute     []*eskip.Clone
 
-	// tracer
-	tracer ot.Tracer
+	// visibility
+	tracer  ot.Tracer
+	metrics metrics.Metrics
```

Comment:
Why do you decide to use Metrics instead of barebone prometheus?
For stock prometheus metrics we can use labels.

Reply:
Mainly reuse our code. Our code also uses labels, so I don't see the benefit of defining here a large chunk of implementation specific metrics.